cmd/proxygenerator/interceptor.go

```diff
@@ -159,6 +162,25 @@ func NewFailureVisitorInterceptor(options FailureVisitorInterceptorOptions) (grp
 	}, nil
 }
 
+func (o *VisitPayloadsOptions) defaultWellKnownAnyVisitor(ctx *VisitPayloadsContext, p *anypb.Any) error {
```

Comment:
There could be a need in the future to expose this for users to delegate to, but we will wait until that need to expose it